boot/zephyr/targets/96b_carbon.h

```diff
@@ -33,3 +33,6 @@
  * sector in size.
  */
 #define FLASH_AREA_IMAGE_SECTOR_SIZE	0x20000
+
+/* The number of bootable images by MCUBoot */
+#define BOOT_IMAGE_COUNT	1
```

Comment:
Individual target files are going away:

https://github.com/runtimeco/mcuboot/pull/114
https://github.com/runtimeco/mcuboot/pull/140

This information is not target-specific, I don't believe. Where else can this information go? Perhaps it could be added to the Zephyr device tree, the way we have moved the flash layout? Or perhaps it can be a top-level mcuboot configuration?

My feeling is that the Zephyr image is going to need to be modified to describe the new flash partition layout in device tree.